spec.emu

```diff
@@ -602,16 +602,15 @@ contributors: Luca Casonato, Guy Bedford, Nicolò Ribaudo
 
             <emu-alg>
               1. Assert: _state_.[[IsLoading]] is *true*.
-              1. If _module_ is a Cyclic Module Record, _module_.[[Status]] is ~new~, and _state_.[[Visited]] does not contain _module_, <ins>and _phase_ is ~evaluation~, </ins>then
+              1. If _module_ is a Cyclic Module Record, _module_.[[Status]] is ~new~, <del>and</del> _state_.[[Visited]] does not contain _module_<ins>, and _phase_ is ~evaluation~, </ins>then
                 1. Append _module_ to _state_.[[Visited]].
                 1. Let _requestedModulesCount_ be the number of elements in _module_.[[RequestedModules]].
                 1. Set _state_.[[PendingModulesCount]] to _state_.[[PendingModulesCount]] + _requestedModulesCount_.
                 1. <del>For each String _required_ of _module_.[[RequestedModules]], do</del>
                 1. <ins>For each ModuleRequest Record _required_ of _module_.[[RequestedModules]], do</ins>
                   1. If _module_.[[LoadedModules]] contains a Record whose [[Specifier]] is _required_<ins>.[[Specifier]]</ins>, then
                     1. Let _record_ be that Record.
-                    1. <ins>If _required_.[[Phase]] is not ~source~, then</ins>
-                      1. Perform InnerModuleLoading(_state_, _record_.[[Module]]<ins>, ~evaluation~</ins>).
+                    1. Perform InnerModuleLoading(_state_, _record_.[[Module]], <ins>_required_.[[Phase]]</ins>).
```

Comment:
We must always call `InnerModuleLoading`, for the steps about `PendingModulesCount`. The loop to recursively load the dependencies has a _phase_ guard anyway, so it will be skipped for source imports.